endpoints/getting-started-grpc/README.md

```diff
@@ -49,13 +49,10 @@ Cloud account and [SDK](https://cloud.google.com/sdk/) configured.
 
     ```bash
     gcloud endpoints services deploy api_descriptor.pb api_config.yaml
-    # The Config ID should be printed out, looks like: 2017-02-01r0, remember this
 
     # Set your project ID as a variable to make commands easier:
     GCLOUD_PROJECT=<Your Project ID>
 
```

Comment:
This blank line should probably also be removed.

Reply:
I think it's better to have the blank line - for better clarity of the output.